spec/requests/comments_request_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+RSpec.describe CommentsController, type: :controller do
+  let(:user_mike) { create(:user_mike) }
+  let(:micropost) { user_mike.microposts.create(content: 'Lorem ipsum') }
+  let(:save_cmt) { user_mike.comments.create(content: 'Lorem ipsum', micropost_id: micropost.id) }
+  let(:save) { Comment.count }
+
+  context 'is logged in' do
```

Comment:
context thì băt đầu bằng trạng từ
where, when, that....
ví dụ ở trường hợp này 
context 'when user logged in" hoặc context "when user is authorized"